libkbfs/folder_branch_ops.go

```diff
@@ -1849,7 +1847,7 @@ func (fbo *folderBranchOps) finalizeBlocks(bps *blockPutState) error {
 }
 
 // Returns true if the passed error indicates a revision conflict.
-func (fbo *folderBranchOps) isRevisionConflict(err error) bool {
+func isRevisionConflict(err error) bool {
```

Comment:
I think this will conflict with one of my outstanding PRs, just FYI.  May the best man win.